samples/vm_external_ip.yaml

```diff
@@ -22,3 +22,5 @@ spec:
     mode: "whitelist"
     instances:
       - //compute.googleapis.com/projects/test-project/zones/us-east1-b/instances/vm-external-ip-two
+      # you may also provide regex expressions, like:
```

Comment:
Since match_mode has to be changed to allow regular expressions, can we provide a separate sample with regex support?

Reply:
Sorry I missed this. I added example. I have also updated the template file. match_mode was missing there